pkg/istio-agent/agent.go

```diff
@@ -450,26 +451,32 @@ func (a *Agent) initSdsServer() error {
 // It will not return until both workload cert and root cert are generated.
```

Comment:
This comment is no longer accurate I think. Should we preserve old model of retry forever?